api/api.go

```diff
@@ -120,12 +125,36 @@ func (h *httpHandler) initRouter() {
 	}
 }
 
-func makeHandler(ctrl libnetwork.NetworkController, fct processor) http.HandlerFunc {
+const (
+	// only one Accept value for now
+	acceptRegex  = `^application/vnd\.libnetwork\.v\d\.\d\+json$`
```

Comment:
Rectifying my previous comment.
 Reason why I am accepting an empty Accept-Header and defaulting to latest version is because the use case when libnetwork API plug into docker  server.
So that a docker client does not have to fill in libnetwork specific headers when it is requesting against "/network" URLs.

I am still keen to making the change to support the common "application/json" but I am not deprecating the empty one.

BTW, I am expecting that once this gets integrated in docker, more discussion and changes will happen to make the two APIs converge to a more common state.